app/src/main/java/com/hyperether/getgoing/repository/room/entity/DbRoute.java

```diff
@@ -62,6 +62,9 @@ public DbRoute createFromParcel(Parcel in) {
         }
     };
 
+    public DbRoute(int i, int i1, int i2, int i3, String s, int i4, int i5) {
+    }
```

Comment:
what is the purpose of this method? If you added it for no reason please remove it.

Reply:
Deleted